packages/headless/src/controllers/product-listing/category-facet/headless-product-listing-category-facet.ts

```diff
@@ -69,31 +68,17 @@ export function buildCategoryFacet(
   const coreController = buildCoreCategoryFacet(engine, props);
   const {dispatch} = engine;
   const getFacetId = () => coreController.state.facetId;
-
-  const createFacetSearch = () => {
-    const {facetSearch} = props.options;
-
-    return buildCategoryFacetSearch(engine, {
-      options: {
-        facetId: getFacetId(),
-        ...facetSearch,
-      },
-      select: (value) => {
-        dispatch(updateFacetOptions({freezeFacetOrder: true}));
-        dispatch(fetchProductListing()).then(() =>
-          logFacetSelect({facetId: getFacetId(), facetValue: value.rawValue})
-        );
-      },
-    });
-  };
-
-  const facetSearch = createFacetSearch();
-  const {state, ...restOfFacetSearch} = facetSearch;
```

Comment:
You sure you needed to remove that? I think it was useful in not exposing `controller.facetSearch.state` :thinking:

Reply:
+1 that was the intent,

Reply:
:+1: I did this change!